package.json

```diff
@@ -26,21 +26,32 @@
   "homepage": "https://github.com/uberVU/mocha-mugshot-reporter#readme",
   "devDependencies": {
     "babel": "~5.8.23",
-    "babel-loader": "~5.3.2",
-    "chai": "~3.2.0",
+    "babel-core": "~6.13.2",
+    "babel-loader": "~6.2.4",
+    "babel-preset-es2015": "~6.13.2",
+    "babel-preset-react": "~6.11.1",
+    "babel-register": "~6.11.6",
+    "chai": "~3.5.0",
     "chai-mugshot": "~0.2.0",
     "classnames": "~2.2.5",
     "css-loader": "~0.19.0",
     "eslint": "~2.0.0",
     "eslint-plugin-react": "~4.1.0",
+    "jsdom": "~9.4.1",
     "lodash": "~3.10.1",
     "mocha": "~2.3.1",
     "mugshot": "^0.2.0",
     "object-assign": "~4.0.1",
+    "path": "~0.12.7",
     "react": "~0.14.8",
+    "react-addons": "~0.9.1-deprecated",
```

Comment:
`addons.classSet` was used in the `Test.jsx` component. Now I'm using `classNames` and I forgot to remove the addons package 😄  .